dockerfiles/enclave.Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM ghcr.io/edgelesssys/ego-dev:latest
```

Comment:
Do we still need a docker file that builds from main regardless of the branch that it runs on ?

Reply:
Yeah I don't think we do, only usage at the moment is in the azure enclave deployers I think. We should migrate that to do its own checkout and use local build I think. But think we should wait until we see what your orchestration work comes out looking like since that will probably impact.